content/yaml/building-a-native-android-app.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Building a native Android app
+description: Building an Android app with YAML.
+weight: 3
+---
+## Building an Android app
+
+The necessary command for building native Android application goes under `scripts` in the [overall architecture](../yaml/yaml/#template) in the `codemagic.yaml` file. For Android (built with gradle), the script looks like this:
+
+    - ./gradlew bundleDebug
```

Comment:
shouldn't it just be ./gradlew build? @BondarenkoStas

Reply:
changed to `build` in 420381a, but it doesn't matter at all, just depends on what users want to do. If they want app bundle - then bundleDebug/bundleRelease, if they want unsigned apk - then assembleDebug/assembleRelease, if they want signed apk, then build